### ISSUE TITLE
C7000 enclosure powerCurrent reporting 0

### ISSUE THREAD

**Issue by davidvf** [HewlettPackard/POSH-HPOneView#595]
I am using HPEOneView.630 to get the utilization metrics from both Synergy and C7000.
On synergy this is working fine:
```
PS C:\Users\vanfroda> $enc = Get-OVEnclosure -name "Frame_1_CZJ72503N8"
PS C:\Users\vanfroda> $enc | Show-OVUtilization

Name               PowerCurrent AmbientTemperature LastUpdated
----               ------------ ------------------ -----------
Frame_1_CZJ72503N8 3,974 W      68 °F              09-Feb-22 19:35:00

PS C:\Users\vanfroda> $metric = $enc | Show-OVUtilization
PS C:\Users\vanfroda> $metric.PowerCurrent
Watts
-----
 3974

PS C:\Users\vanfroda> $metric.PowerAverage
Watts
-----
 1478
```

On C7000 enclosure, powerCurrent is always reporting 0
```
PS C:\Users\vanfroda> $enc = Get-OVEnclosure -name "Openlab_prod_C7000"
PS C:\Users\vanfroda> $enc | Show-OVUtilization
Name               PowerCurrent AmbientTemperature LastUpdated
----               ------------ ------------------ -----------
Openlab_prod_C7000 0 W          64 °F              09-Feb-22 19:35:00


PS C:\Users\vanfroda> $metric = $enc | Show-OVUtilization
PS C:\Users\vanfroda> $metric.PowerCurrent
Watts
-----
    0

PS C:\Users\vanfroda> $metric.PowerAverage
Watts
-----
 3138
```

**Comment by ChrisLynchHPE** [comment]
This is caused by the Cmdlet not handling the c-Class enclosure correctly.  Current power is not stored with the enclosure API object, like a Synergy enclosure resource does.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 6.30.3188.1900](https://github.com/HewlettPackard/POSH-HPEOneView/releases/tag/v6.30.3188.1900) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPEOneView.630/6.30.3188.1900).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
